rollup.config.js

```diff
@@ -13,5 +13,13 @@ export default {
     file: 'dist/index.mjs',
     sourcemap: true,
   },
-  plugins: [commonjs(), nodeResolve({ browser: true }), terser()],
+  plugins: [
+    commonjs(),
+    nodeResolve({
+      // priority increases from left to right
```

Comment:
actually, this is not true - the priority depends on the order in which those conditions appear in `package.json`. I got it wrong cause `@rollup/plugin-node-resolve` is passing this internally as an array, instead of converting this to a Set. So I have thought that the order of the items in this array matters - but it doesn't:
```suggestion
```